src/InputWithLabel.jsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+const InputWithLabel = ({todoTitle, handleTitleChange, children}) => {
+  const inputRef = React.useRef();
+
+  React.useEffect(() => {
+    inputRef.current.focus();
+  });
+
+  return (
+    <>
+      <label htmlFor="todoTitle">{children}</label>
+      <input id="todoTitle" name="title" type="text" value={todoTitle} onChange={handleTitleChange} ref={inputRef}/>
```

Comment:
When there are multiple props, they should start on a new line for better readability. Like this: 
```
      <input
        id="todoTitle"
        name="title"
        type="text"
        value={todoTitle}
        onChange={handleTitleChange}
        ref={inputRef}
      />
```